<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>music</class>
 <widget class="QMainWindow" name="music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Music</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_main">
       <attribute name="title">
        <string>主界面</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_music">
          <property name="title">
           <string>音乐设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButton_notes">
             <property name="text">
              <string>播放模式</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_speed">
               <property name="text">
                <string>速度调节</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_speed">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber_speed"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButton_piano">
             <property name="text">
              <string>弹奏模式</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_serial">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>串口</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="groupBox_receive">
               <property name="title">
                <string>接收区</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QTextEdit" name="textEdit_receive">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QPushButton" name="pushButton_receive_clear">
                    <property name="text">
                     <string>清除接收</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_time_r">
                    <property name="text">
                     <string>接收戳</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_time_s">
                    <property name="text">
                     <string>发送戳</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_receive16">
                    <property name="text">
                     <string>16进制显示</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_save">
                    <property name="text">
                     <string>保存</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>发送区</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QTextEdit" name="textEdit_send">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QPushButton" name="pushButton_send">
                    <property name="text">
                     <string>发送</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_stop">
                    <property name="text">
                     <string>停止</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_send_clear">
                    <property name="text">
                     <string>清除发送</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="checkBox_send16">
                    <property name="text">
                     <string>16进制发送</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_port_setting">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>0</height>
              </size>
             </property>
             <property name="title">
              <string>串口设置</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label_baud_rate">
                <property name="text">
                 <string>波特率</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_baud_rate"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_stop_bit">
                <property name="text">
                 <string>停止位</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_stop_bit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_data_bit">
                <property name="text">
                 <string>数据位</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_data_bit"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_check_bit">
                <property name="text">
                 <string>校验位</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_check_bit"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_operation">
                <property name="text">
                 <string>串口操作</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="2">
               <widget class="QPushButton" name="pushButton_on_off">
                <property name="text">
                 <string>打开串口</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="pushButton_search_port">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>串口搜索</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_serial_port"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_keymap">
       <attribute name="title">
        <string>键位图</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_keymap">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/pic/images/key.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_setup">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_font_r"/>
    <addaction name="action_font_s"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_setup"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_font_r">
   <property name="text">
    <string>接收区字体设置</string>
   </property>
  </action>
  <action name="action_font_s">
   <property name="text">
    <string>发送区字体设置</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>radioButton_notes</tabstop>
  <tabstop>horizontalSlider_speed</tabstop>
  <tabstop>radioButton_piano</tabstop>
  <tabstop>pushButton_search_port</tabstop>
  <tabstop>comboBox_serial_port</tabstop>
  <tabstop>comboBox_baud_rate</tabstop>
  <tabstop>comboBox_stop_bit</tabstop>
  <tabstop>comboBox_data_bit</tabstop>
  <tabstop>comboBox_check_bit</tabstop>
  <tabstop>pushButton_on_off</tabstop>
  <tabstop>textEdit_send</tabstop>
  <tabstop>pushButton_send</tabstop>
  <tabstop>pushButton_stop</tabstop>
  <tabstop>pushButton_send_clear</tabstop>
  <tabstop>checkBox_send16</tabstop>
  <tabstop>textEdit_receive</tabstop>
  <tabstop>pushButton_receive_clear</tabstop>
  <tabstop>checkBox_time_r</tabstop>
  <tabstop>checkBox_time_s</tabstop>
  <tabstop>checkBox_receive16</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
